library/nsxt_transport_nodes.py

```diff
@@ -838,8 +838,8 @@ def main():
                        managed_by_server=dict(required=False, type='str'),
                        host_credential=dict(required=False, type='dict',
                        username=dict(required=False, type='str'),
-                       password=dict(required=False, type='str'),
-                       thumbprint=dict(required=False, type='str')),
+                       password=dict(required=False, type='str', no_log=True),
+                       thumbprint=dict(required=False, type='str', no_log=True)),
```

Comment:
I am not sure if we need to hide the Thumbprint. I think its available through the API. Perhaps you could emulate the behavior. If we can see it via API, no issues seeing it via log.